src/graphql/queries/Organization.ts

```diff
@@ -90,3 +90,33 @@ export const BSPBALANCE = gql`
     bspbalance
   }
 `;
+
+export const GET_ORGANIZATION_COUNT = gql`
+  query countOrganizations($filter: OrganizationFilter!) {
+    countOrganizations(filter: $filter)
+  }
+`;
+
+export const FILTER_ORGANIZATIONS = gql`
```

Comment:
both are the same - FILTER_ORGANIZATIONS, FILTER_ORGANIZATIONS_NAME, please remove/reuse